src/main/telemetry/srxl.c

```diff
@@ -358,17 +380,19 @@ bool srxlFrameFlightPackCurrent(sbuf_t *dst, timeUs_t currentTimeUs)
 
     timeUs_t keepAlive = currentTimeUs - lastTimeSentFPmAh;
 
+
     if ( (amps != sentAmps) || (mah != sentMah) ||
-         keepAlive > FP_MAH_KEEPALIVE_TIME_OUT ) {
+         (keepAlive > FP_MAH_KEEPALIVE_TIME_OUT) ) {
```

Comment:
Unnecessary parentheses should be avoided: https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md#implementation

Reply:
Yes, of course.